doc/releases/release-notes-2.5.rst

```diff
@@ -173,6 +173,10 @@ Drivers and Sensors
 
 * USB
 
+  * Added support for operate only on the primary image by the DFU class which,
```

Comment:
Do you really mean "primary image" or should it be "primary slot" (or "primary partition") ?